.dprintrc.json

```diff
@@ -27,6 +31,8 @@
   "plugins": [
     "https://plugins.dprint.dev/typescript-0.29.0.wasm",
     "https://plugins.dprint.dev/json-0.7.0.wasm",
-    "https://plugins.dprint.dev/markdown-0.4.0.wasm"
+    "https://plugins.dprint.dev/markdown-0.4.0.wasm",
+    "https://plugins.dprint.dev/rustfmt-0.3.0.wasm",
+    "https://plugins.dprint.dev/yapf-0.1.1.exe-plugin@7550683e5f8b76c4825eec5cf88b5dcbd043ade4019c60d79e989e1e7fb6f4d8"
```

Comment:
**Problem**

The YAPF plugin runs an executable that requires python 2 on the path as `python`. The "Building Deno" [instructions](https://deno.land/manual/contributing/building_from_source#building-deno) says:

> Python 2. Ensure that a suffix-less python/python.exe exists in your PATH and it refers to Python 2, not 3.

However, I think some people might still develop without this and this would cause problems. I've tried to make the plugin a single executable, but I haven't found a python 2 to exe solution with support for unbuffered stdio (the `python -u` flag), which is necessary for the dprint cli to communicate correctly to the process plugin.

I think this may be reason enough to close this PR and to not merge it.

Reply:
I think that's fine - you're expected to have Python on path to build Deno, so I think it can apply to formatter as well